pkg/monitor/dynkeepalived.go

```diff
@@ -87,20 +87,22 @@ func doesConfigChanged(curConfig, appliedConfig *config.Node) bool {
 }
 
 func retrieveBootstrapIpAddr(apiVip string) {
-	var err error
 
-	if gBootstrapIP != "" {
-		return
-	}
 	// we don't need to read the bootstrap IP address for bootstrap node
 	if os.Getenv("IS_BOOTSTRAP") == "yes" {
 		gBootstrapIP = ""
 		return
 	}
-	gBootstrapIP, err = config.GetBootstrapIP(apiVip)
-	if err != nil {
-		log.Debugf("Could not retrieve bootstrap IP: %v", err)
+
+	retrievedIP, err := config.GetBootstrapIP(apiVip)
+	if gBootstrapIP != retrievedIP {
+		if err != nil {
+			log.Infof("Could not retrieve bootstrap IP: %v , retrieved IP: %s", err, retrievedIP)
```

Comment:
I wonder if this should be debug. Isn't it going to happen every time the monitor runs after the bootstrap goes down?

Also a nit: retrievedIP will always be "" here. I don't think we need to include it.

Reply:
Hmm, this log should happen only when the current bootstrap IP (retrievedIP) is different from the previous bootstrap IP ( gBootstrapIP), so I assume we should see this line twice on the logs.